Library/Styles/CommentCellStyles.swift

```diff
@@ -27,6 +27,18 @@ public let commentBodyTextViewStyle: TextViewStyle = { textView in
     |> \.dataDetectorTypes .~ .link
 }
 
+// Internationalize
+
+public let collaboratorAuthorBadgeStyle: PaddingLabelStyle = { label in
```

Comment:
Can we pass the text as an argument and eliminate the need for a separate style here?